### PR TITLE
Webchat header

### PR DESCRIPTION
**Pull request by pedroespindula**
**Feature/Bug description:** Adds header and children components

**Solution:** Create one major header component and three children ones:
- logo: Responsive logo component;
- icon-button: Icon button that uses material design components;
- user-description: User showcase;

**TODO/FIXME:** n/a

### CODE REVIEW

File: webchat/styles/custom.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        display: none;
+    }
+}
+
+.no-padding {
+    padding: 0 !important;
+}
+
+.no-margin {
+    margin: 0 !important;
```
Comment (Luiz-FS) [inline webchat/styles/custom.css]: use !important is really necessary?

File: webchat/styles/custom.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+@media screen and (min-width: 1280px) {
+    .hide-on-desktop {
+        display: none;
+    }
+}
+
+.no-padding {
+    padding: 0 !important;
```
Comment (Luiz-FS) [inline webchat/styles/custom.css]: use !important is really necessary?

File: webchat/components/user-description/user-description.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+.user-text {
+    align-content: center;
+    align-self: center;
+    display: flex;
+    flex-direction: column;
+    padding: 0;
+    margin: 0;
+}
+
+.user-name {
```
Comment (Luiz-FS) [inline webchat/components/user-description/user-description.css]: Remove empty class.

File: webchat/components/header/ecis-header.component.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        bindings: {
+            className: "@",
+            style: "@",
+            ariaLabel: "@",
+            mdColors: "@",
+            user: "<"
+        },
+    });
+
+    function headerController ($mdSidenav) {
```
Comment (JuliePessoa) [inline webchat/components/header/ecis-header.component.js]: use inline array annotation here and in all dependency injectors
https://docs.angularjs.org/guide/di#inline-array-annotation

**Pull request merged**